pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -88,6 +88,9 @@ type NodeResourcesFitArgs struct {
 	// with "example.com", such as "example.com/aaa" and "example.com/bbb".
 	// A resource group name can't contain '/'.
 	IgnoredResourceGroups []string
+
+	// ScoringStrategy selects the node resource scoring strategy.
```

Comment:
This is an internal type that we expect to always exist, change it to non-reference.

Reply:
```ScoringStrategy ScoringStrategy```